Sources/SwiftFormat/PrettyPrint/PrettyPrint.swift

```diff
@@ -607,21 +528,19 @@ public class PrettyPrinter {
       }
 
       self.disabledPosition = nil
-      writeRaw(text)
-      if text.hasSuffix("\n") {
-        isAtStartOfLine = true
-        consecutiveNewlineCount = 1
-      } else {
-        isAtStartOfLine = false
-        consecutiveNewlineCount = 0
-      }
+      outputBuffer.writeVerbatimAfterEnablingFormatting(text)
 
     case .disableFormatting(let newPosition):
       assert(disabledPosition == nil)
       disabledPosition = newPosition
     }
   }
 
+  private func shouldBreak(_ length: Int) -> Bool {
```

Comment:
I wonder if this argument should have a default value of 1? Then the cases above where we're checking if we're already at or past the end of the line can just write `shouldBreak()` instead of `shouldBreak(1)`; the latter feels a little "magical".

Continuing down that line of thinking, maybe an even better name for this would be `canFit`. Then the call sites would read like `canFit(length)` where we're looking for a specific count, or `canFit()` for "can anything fit".

WDYT?

Reply:
Makes total sense to me, I've updated with these changes.